id-repository/id-repository-credentials-feeder/src/test/java/io/mosip/idrepository/credentialsfeeder/step/test/CredentialsFeedingWriterTest.java

```diff
@@ -48,6 +49,7 @@
 @WebMvcTest
 @Import(EnvUtil.class)
 @ActiveProfiles("test")
+@Ignore
```

Comment:
why are we ignoring the full test case files?

Reply:
Fixed it.